src/PaymentPart/Output/FpdfOutput/FpdfOutput.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+
+namespace Sprain\SwissQrBill\PaymentPart\Output\FpdfOutput;
+
+use Sprain\SwissQrBill\PaymentPart\Output\AbstractOutput;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\OutputElementInterface;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\Placeholder;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\Text;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\Title;
+use Sprain\SwissQrBill\PaymentPart\Output\FpdfOutput\Template\QrBillFooter;
+use Sprain\SwissQrBill\PaymentPart\Output\OutputInterface;
+use Sprain\SwissQrBill\PaymentPart\Translation\Translation;
+use Sprain\SwissQrBill\QrBill;
+use Sprain\SwissQrBill\QrCode\QrCode;
+
+final class FpdfOutput extends AbstractOutput implements OutputInterface
+{
+    /** @var QrBillFooter */
+    private $fpdf;
+
+    /** @var float */
+    private $amountLS = 0;
+
+    public function __construct(
+        QrBill $qrBill,
+        string $language,
+        QrBillFooter $fpdf
```

Comment:
Do you think we could add a `$offsetX` and `$offsetY` like `TcPdfOutput` does and consider it in all positionings? This would make minor positioning adjustments much simpler for the developer.

Reply:
Yes I can implement this